showcase.json

```diff
@@ -2853,5 +2853,25 @@
   "twitter": "https://twitter.com/221BravoApp",
   "youtube": "",
   "submittableId": "36556639"
+},
+  {
+  "name": "Tali AI ",
+  "description": "Tali is a Discord Support bot that answers common developer queries.",
+  "website": "https://trytali.com ",
+  "logo": "/img/showcase/tali-ai_logo.webp",
+  "video": "/img/showcase/tali-ai_video.mp4",
+  "display": "Normal",
+  "id": "tali-ai",
+  "oneLiner": "Tali is a Discord Support bot that answers common developer queries.",
+  "stats": "100k+ devs spt. ",
```

Comment:
Can you please advise what does `spt.` mean?

Reply:
Hey Jennifer - spt. is supposed to be 'supported'. I was working with the character constraints. 

If I can manually change it in the JSON, happy to do so.

Reply:
Thank you for the clarification. Would you be open to just mentioning `100k+ devs`?

Reply:
Yeah that's fine!

Reply:
Thank you for updates and responses! :)